libs/dh/message-archive/feature-log-search/src/lib/dh-message-archive-log-search.component.html

```diff
@@ -145,8 +112,9 @@ <h1>{{ transloco('title') }}</h1>
         />
       </watt-form-field>
 
+      <!-- Date to -->
       <watt-form-field size="normal">
```

Comment:
suggestion: the default size of `watt-form-field` is "normal" so you don't have to specify it.